cg/apps/housekeeper/hk.py

```diff
@@ -273,7 +277,7 @@ def get_latest_bundle_version(self, bundle_name: str) -> Version | None:
         """Get the latest version of a Housekeeper bundle."""
         last_version: Version = self.last_version(bundle_name)
         if not last_version:
-            LOG.warning(f"No bundle found for {bundle_name} in Housekeeper")
+            LOG.debug(f"No bundle found for {bundle_name} in Housekeeper")
```

Comment:
This one I'm unsure of... Do you think it should warn if no bundle exists?

Reply:
I am confused about the whole method tbh. So we have a wrapper for `self.last_version` but what is the difference between the two apart from the added logging?

Probably out of scope but I do not think there is a need to have both methods present. If we are keeping both, I would say we should log a warning if that is the only difference between the methods.

Reply:
I assume there is proper error handling with relevant logging down the line if the method expects anything else than `None`